### PR TITLE
iOS Landscape Rotation

### PR DESCRIPTION
**Pull request by JoelPickin**
### Description of Change ###

- Register when the device is rotated by the OnElementPropertyChanged property of Height and Width. 
- When rotated set the bounds of the screen.
- When rotated set the camera orientation of the capture connection.

### Bugs Fixed ###
This previously was a bug I raised myself but was understandably changed to a feature-request.

https://github.com/xamarin/XamarinCommunityToolkit/issues/894

### API Changes ###

Added: 
 
- Added method in FormsCameraView.ios.cs: GetViewOrientation - Returns the AVCaptureVideoOrientation of the device.
- Added method in FormsCameraView.ios.cs: SetCameraOrientation - Sets the previewLayer.Connection.VideoOrientation to the AVCaptureVideoOrientation from GetViewOrientation and then assigned the previewLayer.Connection to captureConnection.

Changed:

 - In method InitializeCamerain FormsCameraView.ios.cs added the line previewLayer.Connection.VideoOrientation = GetViewOrientation(); to set the correct orientation when the camera is being initialized.
 - In CameraViewRenderer.ios.cs - Added cases to switch in method OnElementPropertyChanged to check when Height & Width are changed ti trigger Control.SetBounds(Element.Width, Element.Height); & Control.SetCameraOrientation();
 
### Behavioral Changes ###

Rotating the device from landscape to the opposite landscape flips the camera view as the renderer still doesn't register the difference in rotation change. Rotating from portrait to landscape, then to portrait and landscape again works fine. 

Images are captured in landscape also.

### PR Checklist ###

- [ ] Has tests (if omitted, state reason in description)
- [x] Has samples (if omitted, state reason in description) - Works in CameraView sample app
- [x] Rebased on top of main at time of PR
- [x] Changes adhere to coding standard

### CODE REVIEW

**Comment by pictos** [comment]
@JoelPickin, can you rebase your PR against develop?

**Comment by JoelPickin** [comment]
@pictos have done now. Thanks.

**Comment by AndreiMisiukevich** [comment]
> @pictos have done now. Thanks.

something went wrong.
it seems you changed base branch on Github but didn't rebase in git

**Comment by JoelPickin** [comment]
> > @pictos have done now. Thanks.
> 
> something went wrong.
> it seems you changed base branch on Github but didn't rebase in git

Sorry about this. I thought I had rebased correctly. I am having some issues building at the moment. I have tried pulling down a fresh version to look at trying again, but I am getting a lot of UWP errors.

File: azure-pipelines.yml [hunk 1]
```diff
@@ -116,19 +113,19 @@ jobs:
         condition: startsWith(variables['Build.SourceBranch'], 'refs/tags/')
       # restore, build and pack the packages
       - task: MSBuild@1
-        displayName: Build Xamarin.CommunityToolkit.csproj
+        displayName: Build Solution
         inputs:
-          solution: $(PathToCommunityToolkitCsproj)
+          solution: $(PathToCsproj)
```
Comment (brminnick) [inline azure-pipelines.yml:118]: Please revert all changes to `azure-pipelines.yml`

**Review by brminnick — CHANGES_REQUESTED** [review]

**Comment by JoelPickin** [comment]
I'm going to close this PR and add my changes in a new branch and push that up as I'm not sure what the develop/rebase has done and feel it would be better starting fresh from the most recent.

**Comment by JoelPickin** [comment]
New PR https://github.com/xamarin/XamarinCommunityToolkit/pull/1224

**Pull request closed without merging**